src/activities/menu/ConfigurationItem.qml

```diff
@@ -404,6 +413,58 @@ Item {
                 }
             }
         }
+        
+        Flow {
```

Comment:
I agree with Holger. If automatic download, we don't show the dialog and start the download else we show it. At the end of the download, we enable/register the music.
Do not touch the wordset for now, it should be done in another pr